### ISSUE TITLE
bug: swipe to go back in iOS is activating instead of opening the menu

### ISSUE THREAD

**Issue by brandyscarney** [ionic-team/ionic-framework#18879]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

1. Run the conference app using version `4.5.0`: https://github.com/driftyco/ionic-conference-app
2. Open the schedule page in Chrome (on desktop) and in Safari on iOS and try to swipe the menu open

This is the correct behavior, repeat the steps using `latest` (`4.6.2` at time of writing) and notice that it swipes back to the tutorial page. I was able to reproduce this in Chrome.

**Expected behavior:**

It should work like `4.5.0`.

**Steps to reproduce:**

See above.

**Related code:**

```
insert short code snippets here
```

**Other information:**

**Ionic info:**

```
insert the output from ionic info here
```

**Comment by liamdebeasi** [comment]
This will be fixed in Ionic 4.7.0 🎉

**Comment by liamdebeasi** [comment]
I can't reproduce this anymore using the latest version of Ionic 6 + the Angular conference app. Closing.
